internal/worker_deployments.go

```diff
@@ -0,0 +1,58 @@
+package internal
+
+import (
+	"fmt"
+	"strings"
+
+	deploymentpb "go.temporal.io/api/deployment/v1"
+)
+
+// Represents the version of a specific worker deployment.
+//
+// NOTE: Experimental
+//
+// Exposed as: [go.temporal.io/sdk/client.WorkerDeploymentVersion], [go.temporal.io/sdk/worker.WorkerDeploymentVersion]
```

Comment:
Why is this exposed twice?

Reply:
Just because it applies in both packages and that seems kinda useful. It show up in various client responses, but also as a parameter to worker options. If you'd prefer it in just one, no problem, I suppose client?